examples/va/nfv/sriov/edpm/values.yaml

```diff
@@ -157,8 +94,73 @@ spec:
         # edpm firewall, change the allowed CIDR if needed
         edpm_sshd_configure_firewall: true
         edpm_sshd_allowed_ranges:
-          - 192.168.122.0/24 # CHANGEME
+          - 192.168.122.0/24
         # SELinux module
         edpm_selinux_mode: enforcing
         # SRIOV settings
-        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: 'sriov-phy4:eno4' # CHANGEME
+        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: 'sriov-phy4:eno4'
+    nodes:
+      edpm-compute-0:
+        ansible:
+          ansibleHost: 192.168.122.100
+        hostName: edpm-compute-0
+        networks:
+        - defaultRoute: true
+          fixedIP: 192.168.122.100
+          name: CtlPlane
+          subnetName: subnet1
+        - name: InternalApi
+          subnetName: subnet1
+        - name: Storage
+          subnetName: subnet1
+        - name: Tenant
+          subnetName: subnet1
+      edpm-compute-1:
+        ansible:
+          ansibleHost: 192.168.122.101
+        hostName: edpm-compute-1
+        networks:
+        - defaultRoute: true
+          fixedIP: 192.168.122.101
+          name: CtlPlane
+          subnetName: subnet1
+        - name: InternalApi
+          subnetName: subnet1
+        - name: Storage
+          subnetName: subnet1
+        - name: Tenant
+          subnetName: subnet1
+    services:
+    - bootstrap
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - reboot-os
+    - libvirt
+    - ovn
+    - nova-custom-sriov
+    - neutron-sriov
+    - neutron-metadata
+  nova:
+    compute:
+      conf: |
+        [compute]
+        cpu_shared_set = 0-3,24-27
+        cpu_dedicated_set = 8-23,32-47
+        reserved_host_memory_mb = 4096
+    migration:
+      ssh_keys:
+        private: |
+          CHANGEME4
+          CHANGEME4
+          CHANGEME4
+        public: |
+          CHANGEME5
+          CHANGEME5
+          CHANGEME5
+    pci:
+      conf: |
+        [pci]
+        passthrough_whitelist = { "devname":"eno4", "physical_network":"sriov-phy4", "trusted":"true" }
```

Comment:
as noted on https://github.com/openstack-k8s-operators/architecture/pull/62

device name was deprecated in 17.1 downstream and will be unsupported in our downstream product in 18 and 19.
to get the same behaviour you should use the PCI adress of the PF
with the vendor_id and product_id of the VFs.

device_spec = {"vendor_id":"a2d6",
                   "product_id":"15b3",
                   "address": "0000:82:00.0",
                   "physical_network":"physnet1",
               }

this will make all VFs of the PF consumable.

as part of a move to inclusive naming passthrough_whitelist
was replace with device_spec and is no longer supported in 18

we should not use passthrough_whitelist in any future docs or in the va.
while its technically still functional we expect the PCI section to be updated as part of the adoption to use device_sepc instead.

some newer features in osp 19 will only work with device_spec and do not support devname.

we have recommend against using devname for many upstream and downstream release  at this point so hopefully most of our customer have already stopped using device name at this point

Reply:
Thank you for the update.  Is it still possible to use the current keys in the example, i.e:

```
{ "devname":"eno4", "physical_network":"sriov-phy4", "trusted":"true" }
```

?

Reply:
Ah, I see.  Thank you for the further clarification.  We will use `vendor_id`, `product_id` and `address` instead.

Reply:
Fixed